src/v2/BaseLoginRouter.js

```diff
@@ -31,10 +31,14 @@ import AppState from './models/AppState';
 import idx from 'idx';
 
 const startLoginFlow = (settings) => {
+  let stateHandle = sessionStorage.getItem('stateTokenInUse');
```

Comment:
Would it make sense to keep this not as clear as possible for security reasons ? for eg. just `widget-state-handle` ?

Reply:
security is worth considering. But I also think it is nice to namespace all our storage so it is clear to our customers and support what we own/manage vs. what their app is doing. In this case I think it would be better to keep the `okta-` prefix, matching other storage that we have for things like tokens (which are more sensitive from a security perspective)